### PR TITLE
Add image of guide to “what to expect” page

### PR DESCRIPTION
**Pull request by alanmoo**
Closes https://github.com/MozillaFoundation/Mozfest2016_production/issues/221

### CODE REVIEW

File: less/pages/expect.less [hunk 1]
```diff
@@ -15,6 +15,32 @@
...
   }
+  p img {
+    max-width: 100%;
+    height: auto;
+  }
+  .guide-image-wrapper{
+    position: relative;
+    display: block;
+  }
+  .guide-image-wrapper:after{
```
Comment (mmmavis) [inline less/pages/expect.less]: awesome! love how this is implemented! 👍

Comment (mmmavis) [inline less/pages/expect.less]: nitpick - add a space before `{`?

File: less/pages/expect.less [hunk 2]
```diff
@@ -15,6 +15,32 @@
...
+    height: 100%;
+    top: 0;
+    left: 0;
+    transition: opacity .25s ease;
+    color: white;
+    text-align: center;
+    padding-top: 30%;
+    font-size: 1.5em;
+  }
+  .guide-image-wrapper:hover::after{
```
Comment (mmmavis) [inline less/pages/expect.less]: nitpick - add a space before `{`?

File: less/pages/expect.less [hunk 3]
```diff
@@ -15,6 +15,32 @@
     max-width: 550px;
     margin: 0 auto;
   }
+  p img {
+    max-width: 100%;
+    height: auto;
+  }
+  .guide-image-wrapper{
```
Comment (mmmavis) [inline less/pages/expect.less]: oh and missing space here

**Comment by alanmoo** [comment]
Added

**Comment by mmmavis** [comment]
🎉

**Pull request merged**